x-pack/plugins/ml/server/routes/schemas/data_visualizer_schema.ts

```diff
@@ -12,13 +12,17 @@ export const indexPatternTitleSchema = schema.object({
   indexPatternTitle: schema.string(),
 });
 
+const runtimeMappingsSchema = schema.maybe(schema.any());
```

Comment:
I reckon `any` isn't safe, because we probably don't want it to accept numbers or strings. And based on `RuntimeFields` we can provide a custom validator. Consider the following schema:
```suggestion
const runtimeMappingsSchema = schema.maybe(schema.object(
            {},
            {
              unknowns: 'allow',
              validate: (v: object) => {
                if (Object.values(v).some((o) => !isRuntimeField(o))) {
                  return 'Your error message about invalid runtime definition';
                }
              },
            }
          ),);
```

Reply:
Updated here [`b68e736` (#95252)](https://github.com/elastic/kibana/pull/95252/commits/b68e7366c3f40de28280ba378449f074039923b6)